src/Ledger/Chain.lagda

```diff
@@ -60,6 +60,7 @@ private variable
 
 instance
   _ = +-0-monoid
+  _ = +-0-commutativeMonoid
```

Comment:
Do we actually need commutativity anywhere?

Reply:
Yeah, it's used in `totWithdrawals = Σᵐᵛ[ x ← trWithdrawals ᶠᵐ ] x`. Without commutativity this is undefined, because the definition can't specify in which order this sum should happen.